lib/helper.ts

```diff
@@ -43,22 +43,31 @@ export interface FileStream extends NodeJS.ReadableStream {
 }
 
 // custom type guards
-function isFileObject(obj: any): obj is FileObject {
+export function isFileObject(obj: any): obj is FileObject {
   return Boolean(obj && obj.value);
 }
 
 function isFileStream(obj: any): obj is FileStream {
   return obj && isReadable(obj) && obj.path;
 }
 
+export function isFileParamAttributes(obj: any): obj is FileParamAttributes {
```

Comment:
```ts
obj is FileParamAttributes
```

Cool, I've never seen this! Does this imply a Boolean return value? Did you find any docs/posts about this and if so, could you link me? Looks interesting